src/test/java/com/puppycrawl/tools/checkstyle/AbstractXmlTestSupport.java

```diff
@@ -73,6 +99,15 @@ protected static void verifyXml(String expectedOutputFile,
         verifyXmlNode(expectedDocument, actualDocument, "/", ordered);
     }
 
+    /**
+     * Verifies if xml nodes in actual generated xml document and expected xml document match
+     * or not.
+     *
+     * @param expected the expected xml node. A {@link Node} interface.
```

Comment:
Here and below: please use one sentence.

Reply:
Done!